post-processor.go

```diff
@@ -285,7 +289,8 @@ func (p *PostProcessor) putManifest(manifest *Manifest) error {
 }
 
 func generateS3Url(region, bucket, key string) string {
-	return fmt.Sprintf("https://s3-%s.amazonaws.com/%s/%s", region, bucket, key)
+	regionString = fmt.Sprintf("-%s", region)
+	return fmt.Sprintf("https://s3%s.amazonaws.com/%s/%s", regionString, bucket, key)
```

Comment:
if `region` is the empty string, won't this generate `https://s3-.amazonaws.com/...`? Is that correct?